src/applications/Machine.java

```diff
@@ -31,24 +31,17 @@ public int getTotalWait() {
         return totalWait;
     }
 
-    public void setTotalWait(int totalWait) {
-        this.totalWait = totalWait;
-    }
-
     public int getNumTasks() {
         return numTasks;
     }
 
-    public void setNumTasks(int numTasks) {
-        this.numTasks = numTasks;
-    }
-
-    public Job getActiveJob() {
-        return activeJob;
-    }
 
-    public void setActiveJob(Job activeJob) {
-        this.activeJob = activeJob;
+    void processJob(EventList eList, int theMachine, int timeNow) {
```

Comment:
```suggestion
    private void processJob(EventList eList, int machine, int timeNow) {
```

Reply:
I like this function. However, `process` is a pretty ambiguous word. It would be nice to see a more specific function name. It definitely cleans up `changeState()`.